be/io.c

```diff
@@ -76,10 +76,12 @@ static void be_io_part_init(struct m0_be_io_part *bip,
 	bip->bip_bio = bio;
 	m0_stob_io_init(&bip->bip_sio);
 	m0_clink_init(&bip->bip_clink, be_io_cb);
+	m0_clink_add_lock(&bip->bip_sio.si_wait, &bip->bip_clink);
```

Comment:
Why are these changes here?

Reply:
I did not know

Reply:
I have not edited

Reply:
That's why I recommended you to avoid force-pushes. Please, rebase on the latest main branch and remove all these changes.

And don't do force pushes anymore!

Reply:
It happened after your merging i believe

Reply:
No, it happened after my merging AND your force pushing!

Don't force push anymore, please.